server/setup.go

```diff
@@ -16,6 +16,10 @@ const (
 	defaultAdminUsername = "sysadmin"
 	defaultAdminPassword = "Sys@dmin123"
 	defaultAdminEmail    = "success+sysadmin@simulator.amazonses.com"
+
+	defaultUserUsername = "user"
+	defaultUserPassword = defaultAdminPassword
```

Comment:
Hmm, I think it may be good to use a slightly different password for the user account

Reply:
I though about it, but used the same because thought that people may have muscle memory typing the other, but it's most likely they just copy and paste the output. Changed it to something different.